DataProcessing/series.py

```diff
@@ -0,0 +1,62 @@
+import pandas as pd
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+import json
+import re
+import csv
+import os
+
+class Series:
+    OUTPUT_DIR = "../output/alternative/bls/"
```

Comment:
The output folder shouldn't be 'output', but and env that can default to `'/temp-output-directory'`, see other data sources
Please merge the processing script into a single process.py file, so it's compatible with current CI